tools/orca/src/models/model_markov.rs

```diff
@@ -0,0 +1,143 @@
+use std::{slice::Iter, time::Duration};
+
+use crate::{
+    error::Error,
+    model::{self, ActorModel, Transition, TransitionAction, TransitionResult},
+    run::EventRecord,
+    state::Person,
+};
+use async_trait::async_trait;
+use kanidm_client::KanidmClient;
+use ndarray::{arr2, Array1, Array2, ArrayView2};
+use rand::{
+    distributions::{self, Distribution},
+    rngs::StdRng,
+    Rng, SeedableRng,
+};
+use statrs::distribution::{Multinomial, Normal};
+use strum::EnumCount;
+use strum_macros::EnumCount;
+
+// The idea is to have a multinomial distribution for each state that represents
```

Comment:
I think you went "too hard too fast" on the stats side.

I think you need to back up a bit and make some simpler models. Stuff that just does a linear set of actions first in a loop. Think "login, read attr, set attr, logout, repeat". I think we need those in place first because that gives the foundation of "how to create the entries in the db". 

From there we need ways to direct those models based on attributes of the person. If in group X do Y, else do Z. again, in a loop.

And then once we have that, then we can start to add in randomisation into the actions of the persons. Remember also some of these linear-loop models are good for other tests. Consider a model of login and loop on infinite searches. That's a search benchmark now! 

So I think backing out of this a bit and really stepping back to create simpler models first and then work up will be more valuable.

Reply:
> I think you went "too hard too fast" on the stats side.
> 
> I think you need to back up a bit and make some simpler models. Stuff that just does a linear set of actions first in a loop. Think "login, read attr, set attr, logout, repeat". I think we need those in place first because that gives the foundation of "how to create the entries in the db".
> 

I got this working, now onto the group part!

Reply:
> From there we need ways to direct those models based on attributes of the person. If in group X do Y, else do Z. again, in a loop.
> 
I realized this could be done in two different ways. That is: do we want to be able to dynamically change a person's behavior by changing their groups, which means querying kani each time, or is it enough to just use whatever group we assigned at the start and we don't care about possible changes?